src/js/outreach/entities-service/entities/outreach.js

```diff
@@ -0,0 +1,63 @@
+import Store from 'backbone.store';
+import BaseCollection from 'js/base/collection';
+import BaseModel from 'js/base/model';
+import fetcher, { handleJSON } from 'js/base/fetch';
+
+const TYPE = 'outreach';
+
+const _Model = BaseModel.extend({
+  type: TYPE,
+  urlRoot: '/api/outreach',
+  registerPatient(attributes) {
+    return fetcher('/api/outreach', {
+      method: 'POST',
+      data: JSON.stringify({
+        data: {
+          type: 'patients',
+          attributes,
+        },
+      }),
+    })
+      .then(handleJSON);
+  },
+  requestOtpCreation() {
+    return fetcher('/api/outreach/otp', {
+      method: 'POST',
+      data: JSON.stringify({
+        data: {
+          type: 'patient-actions',
```

Comment:
is this correct?  I don't see `patient-actions` anywhere in the openapi documentation.. we're sure this shouldn't just be `return this.fetch({ url: '/api/outreach/otp' });`?  If this needs to be `patient-actions` then this isn't an outreach model and should just be it's own radio request outside of this model.

Reply:
It's here in the docs:

![Screenshot 2023-08-24 at 10 03 25 AM](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/b49eacfc-92aa-45b1-b6ec-5fe37ba1909e)

@shadowhand is this correct that the `/api/outreach/otp` endpoint has a `type: patient-actions`?